selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/ImportAndValidateEclipseCheProjectTest.java

```diff
@@ -115,26 +108,23 @@ public void checkImportAndResolveDependenciesEclipceCheProject() {
 
     // then open files
     // open a java file
-    projectExplorer.quickRevealToItemWithJavaScript(
-        String.format("%s/%s", PROJECT_NAME, PATH_TO_JAVA_FILE));
-    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_JAVA_FILE));
+    projectExplorer.quickRevealToItemWithJavaScript(PATH_TO_JAVA_FILE);
+    projectExplorer.openItemByPath(PATH_TO_JAVA_FILE);
     editor.waitActive();
 
     // open a xml file
-    projectExplorer.quickRevealToItemWithJavaScript(
-        String.format("%s/%s", PROJECT_NAME, PATH_TO_POM_FILE));
-    projectExplorer.openItemByPath(String.format("%s/%s", PROJECT_NAME, PATH_TO_POM_FILE));
+    projectExplorer.quickRevealToItemWithJavaScript(PATH_TO_POM_FILE);
```

Comment:
this please repeats several times:
```
 projectExplorer.quickRevealToItemWithJavaScript( ... );
 projectExplorer.openItemByPath( ... );
```
what about to create a private method in the test or public method in the "ProjectExplorer" which will encapsulate this two methods, something like that:
```private void quickRevealToItemWithJavaScriptAndOpenFile(String pathToItem){
    projectExplorer.quickRevealToItemWithJavaScript(pathToItem);
    projectExplorer.openItemByPath(pathToItem);
  }
```

Reply:
I agree. I think it is better to do in the test.